### DISCUSSION TITLE
what is the diffrences between jedi-lsp and python-lsp

### DISCUSSION THREAD

**Discussion by ogios** [python-lsp/python-lsp-server#427] [General]
i'm new to neovim and currently using jedi-lsp, can i migrate to python-lsp?
and what it the diffrences between these two?

**Comment by ccordoba12** [comment]
That's mentioned in their Github repositories and/or Readmes:

* Jedi-lsp:

    > A Python language server exclusively for Jedi. If Jedi supports it well, this language server should too.

* Python-lsp:

    > The base language server requires [Jedi](https://github.com/davidhalter/jedi) to provide Completions, Definitions, Hover, References, Signature Help, and Symbols. If the respective dependencies are found, the following optional providers will be enabled ...

So, python-lsp provides more functionality than jedi-lsp.

And [Jedi](https://github.com/davidhalter/jedi) is the most complete code completion library written in Python.
